### PR TITLE
Add support in grails-gradle-plugin for a pathingJar

### PR DESCRIPTION
**Pull request by jameskleeh**
Will only execute if on windows and configuration is set to true

### CODE REVIEW

File: grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy [hunk 1]
```diff
@@ -407,4 +413,63 @@ class GrailsGradlePlugin extends GroovyPlugin {
...
+            ConfigurationContainer configurations =  project.configurations
+            Configuration runtime = configurations.getByName('runtime')
+            Configuration console = configurations.getByName('console')
+
+            Jar pathingJar = createPathingJarTask(project, "pathingJar", runtime)
+            Jar pathingJarCommand = createPathingJarTask(project, "pathingJarCommand", runtime, console)
+
+            GrailsExtension grailsExt = project.extensions.getByType(GrailsExtension)
+
+            if (grailsExt.pathingJar && Os.isFamily(Os.FAMILY_WINDOWS)) {
```
Comment (graemerocher) [inline grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy]: Shouldn't the tasks only be created if `pathingJar` is true?

Comment (Schlogen) [inline grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy]: Ya I originally had it that way. I wasn't sure if there was a possible use case where someone would want to run it standalone or have a custom task that depends on it...

**Review by verglor — COMMENTED** [review]
[delete]

File: grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy [hunk 2]
```diff
@@ -432,13 +430,6 @@ class GrailsGradlePlugin extends GroovyPlugin {
...
                 }
             }
-
-            // assetCompile will be execute for all Jar-type tasks
-            // (see https://github.com/bertramdev/asset-pipeline/blob/master/asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetPipelinePlugin.groovy#L85)
-            // exclude the assets from pathing jar
-            task.exclude {
-                it.file.absolutePath.startsWith(assets.absolutePath)
-            }
         }
```
Comment (jako512) [inline grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy]: Why this workaround for https://github.com/bertramdev/grails-asset-pipeline/issues/314 has been removed ?

**Pull request merged**